catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -436,6 +446,11 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 				.show();
 	}
 
+	protected void showMergeAlert(List<T> selectedItems) {
```

Comment:
since this doesnt show any alert, please rename the method, or even better inline the two calls below and delete it.